nmdc_automation/config/workflows/workflows.yaml

```diff
@@ -328,9 +328,9 @@ Workflows:
       description: CheckM for {id}
       name: CheckM statistics report
     - output: final_hqmq_bins_zip
-      data_object_type: Metagenome HQMQ Bins Compression File
```

Comment:
This is backwards, keep Metagenome HQMQ Bins Compression File, remove Metagenome Bins

Reply:
resolved by [696c8f6](https://github.com/microbiomedata/nmdc_automation/pull/253/commits/696c8f6ad69c2ee478295bdca2aec3e952adec46)